bridge-spring/src/main/java/io/github/xanthic/cache/bridge/spring/XanthicSpringCache.java

```diff
@@ -0,0 +1,55 @@
+package io.github.xanthic.cache.bridge.spring;
+
+import io.github.xanthic.cache.api.Cache;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.cache.support.AbstractValueAdaptingCache;
+
+import java.util.concurrent.Callable;
+
+public class XanthicSpringCache extends AbstractValueAdaptingCache {
+	private final String name;
+	private final Cache<Object, Object> cache;
+
+	public XanthicSpringCache(String name, Cache<Object, Object> cache) {
+		super(true);
+		this.name = name;
+		this.cache = cache;
+	}
+
+	@Override
+	public @NotNull String getName() {
+		return name;
+	}
+
+	@Override
+	public @NotNull Object getNativeCache() {
+		return cache;
+	}
+
+	@Override
+	public <T> T get(@NotNull Object key, @NotNull Callable<T> valueLoader) {
+		return null;
```

Comment:
Should use `computeIfAbsent` here, but may need some AtomicReference complexity for the exception case

Reply:
Wanted to do this method last as i was unsure how/when it is used by spring and then forgot about it